wooting-macro-backend/src/lib.rs

```diff
@@ -378,9 +376,15 @@ fn check_macro_execution_efficiently(
 }
 
 impl MacroBackend {
-    #[cfg(not(debug_assertions))]
     /// Creates the data directory if not present in %appdata% (only in release build).
     pub fn generate_directories() {
+        #[cfg(not(debug_assertions))]
+        match std::fs::remove_file(config::LogFileName::file_name()) {
+            Ok(_) => info!("removed old log file"),
+            Err(error) => error!("Couldnt remove old log file!: {}", error),
+        }
```

Comment:
?

Reply:
Must have pulled in from the other branch, no need to remove the log.. Fixed in a02f397